swap_meet/vendor.py

```diff
@@ -1,2 +1,84 @@
 class Vendor:
-    pass
\ No newline at end of file
+    # Set up inventory
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
```

Comment:
Great job avoiding the side effects of having an empty list as the default value for the inventory argument. Just to save a little bit of time, I recommend converting this into a ternary! 
```suggestion
        self.inventory = [] if inventory is None else inventory
```

Reply:
Thank you! I’ve run into issues before with mutable default arguments, so I’ve made it a habit to always check for None and reassign.